test/browser/spider.js

```diff
@@ -27,13 +47,26 @@ function crawl() {
       resourceData,
       queueItem
     ]);
+    var unicodeResourceData = resourceData.toString("utf8");
 
-    resources.push.apply(resources, findSrcSetURLs(
-      resourceData.toString("utf8"),
-      queueItem.url
-    ));
+    URL_FINDERS.forEach(function(findURLs) {
+      resources.push.apply(resources, findURLs(
+        unicodeResourceData,
+        queueItem.url
+      ));
+    });
 
-    return resources;
+    return resources.map(function(url) {
+      // It's possible that some of our URLs might be absolute URLs
+      // based on config.ORIGIN. However, because the server we've
+      // started for crawling is based at a dynamic origin on localhost,
+      // we need to rebase such URLs to be at our dynamic origin, so
+      // that they're actually spidered.
+      if (url.indexOf(config.ORIGIN) === 0) {
```

Comment:
see rationale for this [above](https://github.com/mozilla/teach.webmaker.org/pull/1051/files#r32827149).